### PR TITLE
Restore auto fill form

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by sbrunner** [comment]
@llienher This time I think that's working, can you confirm?

File: src/auth/FormElement.ts [hunk 1]
```diff
@@ -251,6 +247,7 @@ export default class GmfAuthForm extends GmfBaseElement {
                     type="password"
                     class="form-control"
                     name="oldpassword"
+                    autocomplte="current-password"
```
Comment (llienher) [inline src/auth/FormElement.ts:250]:
```suggestion
                    autocomplete="current-password"
```

File: src/auth/FormElement.ts [hunk 2]
```diff
@@ -259,6 +256,7 @@ export default class GmfAuthForm extends GmfBaseElement {
                     type="password"
                     class="form-control"
                     name="newpassword"
+                    autocomplte="new-password"
```
Comment (llienher) [inline src/auth/FormElement.ts:259]:
```suggestion
                    autocomplete="new-password"
```

File: src/auth/FormElement.ts [hunk 3]
```diff
@@ -267,6 +265,7 @@ export default class GmfAuthForm extends GmfBaseElement {
                     type="password"
                     class="form-control"
                     name="newpasswordconfirm"
+                    autocomplte="new-password"
```
Comment (llienher) [inline src/auth/FormElement.ts:268]:
```suggestion
                    autocomplete="new-password"
```

File: src/auth/FormElement.ts [hunk 4]
```diff
@@ -294,6 +293,7 @@ export default class GmfAuthForm extends GmfBaseElement {
                           autocomplete="off"
                           class="form-control"
                           name="otp"
+                          autocomplete="one-time-code"
```
[the remark this answers is not in the document]
Comment (sbrunner) [reply src/auth/FormElement.ts:296]: Yes, it's part of recommendations :-)

File: src/auth/FormElement.ts (hunk 1 above)
Comment (sbrunner) [reply src/auth/FormElement.ts:250]: Oups, thanks :-)

**Comment by llienher** [comment]
@sbrunner Both mobile and mobile_alt forms are broken and the field are not visible anymore.

**Comment by llienher** [comment]
If I'm not wrong, mobile interface used directly the auth-form component, when desktop used the auth-panel that included the form inside, so the error could be there.

**Comment by sbrunner** [comment]
@llienher I just fix it, probably they are not updated now...
and the CI is failing ...

**Review by llienher — APPROVED** [review]
Thanks ! Tested and looks fine ! :-)

**Pull request merged**